src/Sarif.Viewer.VisualStudio/Views/CodeLocations.xaml

```diff
@@ -153,6 +154,23 @@
                 <ContentControl ContentTemplate="{StaticResource AnnotatedCodeLocationCollectionDataGridTemplate}"
                                 Content="{Binding CodeFlows}" />
             </TabItem>
+            <TabItem Visibility="{Binding CallTrees, Converter={StaticResource collectionToVisibility0Converter}}">
```

Comment:
@jericahuang @kschecht Here's where we add the extra tab you see in the screen shot.